app/pb_kits/playbook/pb_label_pill/label_pill.rb

```diff
@@ -14,7 +14,7 @@ class LabelPill
       prop :pill_value
 
       def classname
-        generate_classname("pb_label_pill_kit", variant)
```

Comment:
Passing the `variant` here was unneeded. The styles for this kit [ignore text after its default kit classname](https://github.com/powerhome/playbook/blob/v4.16.0/app/pb_kits/playbook/pb_label_pill/_label_pill.scss#L4). The `variant` is passed along to the Pill kit, but is not used in the LabelPill's classname.  The test is updated accordingly, and screenshots of both sides have been posted.

Reply:
cc @jasoncypret